spec/unit/cli_spec.rb

```diff
@@ -183,19 +190,35 @@
         run_command(["inspect", "--exclude-scope=packages,repositories", example_host])
       end
 
-      it "forwards the --skip-files option to the InspectTask" do
-        expect_any_instance_of(InspectTask).to receive(:inspect_system).
-          with(
-            an_instance_of(SystemDescriptionStore),
-            example_host,
-            example_host,
-            an_instance_of(CurrentUser),
-            Inspector.all_scopes,
-            skip_files: "/foo/bar"
-          ).
-          and_return(description)
+      it "forwards the --skip-files option to the InspectTask as an unmanaged_files filter" do
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
+          _host, _name, _user, _scopes, filter, _options|
+          expect(filter.element_filter_for("/unmanaged_files/files/name").matchers).
+            to include("/foo/bar", "/baz")
+        end.and_return(description)
 
-        run_command(["inspect", "--skip-files=/foo/bar", example_host])
+        run_command(["inspect", "--skip-files=/foo/bar,/baz", example_host])
+      end
+
+      it "the --skip-files option reads a list of excluded files from a file" do
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
```

Comment:
Block argument expression is not on the same line as the block start.<br>Extra space before block parameter detected.